demos/inpage/demo/config/config.testnet.ts

```diff
@@ -15,6 +15,9 @@ const config: ConfigType = {
     (import.meta.env.VITE_RPC_URL as string) ??
     'https://sepolia-rollup.arbitrum.io/rpc',
 
+  // If a default seed phrase is not provided, a user must provide their own.
+  deployerSeedPhrase: (import.meta.env.VITE_RPC_URL as string) ?? '',
```

Comment:
Is `VITE_RPC_URL` the right env var for the seed phrase?

Reply:
Good catch, I changed it in my config locally but forgot to change it in the template config